{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-4508",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/exchange.xforce.ibmcloud.com\/vulnerabilities\/164429",
                    "name": "ibm-qradar-cve20194508-info-disc (164429)",
                    "refsource": "XF",
                    "tags": []
                },
                {
                    "url": "https:\/\/www.ibm.com\/support\/pages\/node\/1170334",
                    "name": "https:\/\/www.ibm.com\/support\/pages\/node\/1170334",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "IBM QRadar SIEM 7.3.0 through 7.3.3 uses weak credential storage in some instances which could be decrypted by a local attacker. IBM X-Force ID: 164429."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2020-01-10T16:15Z",
    "lastModifiedDate": "2020-01-10T16:15Z"
}